CodeGeneration/Sources/generate-swiftsyntax/templates/swiftsyntax/SyntaxCollectionsFile.swift

```diff
@@ -20,7 +20,9 @@ let syntaxCollectionsFile = SourceFileSyntax(leadingTrivia: copyrightHeader) {
     try! StructDeclSyntax(
       """
       \(raw: node.documentation)
-      public struct \(raw: node.kind.syntaxType): SyntaxCollection, SyntaxHashable
+      \(raw: node.documentation.isEmpty ? "" : "///")
+      \(raw: node.grammar)
```

Comment:
Really cool changes! 💙
Maybe it would be cool to use `removedEmptyLines(string:)` to prevent adding empty lines? It could look this: 
```swift
    let docComment = docCommentTrivia(
      from: removedEmptyLines(
        string: """
        \(node.documentation)
        \(node.documentation.isEmpty ? "" : "///")
        \(node.grammar)
        """
      )
    )

    try! StructDeclSyntax(
      """
      \(raw: docComment)
      public struct \(node.kind.syntaxType): SyntaxCollection, SyntaxHashable
      """
    ) {
(...)
```

Reply:
Very good idea 👍🏽